wom/src/main/scala/wom/graph/expression/AnonymousExpressionNode.scala

```diff
@@ -23,4 +26,8 @@ object AnonymousExpressionNode {
 /**
   * An expression node that is purely an internal expression and shouldn't be visible outside the graph
   */
-trait AnonymousExpressionNode extends ExpressionNode
+trait AnonymousExpressionNode extends ExpressionNode {
+  val callNodeReceivingInput: GraphNodeSetter[CallNode]
```

Comment:
Maybe Scala docs are a good idea on this, given our uneasiness over the name?

Reply:
yes